superset-frontend/src/explore/components/controls/MetricControl/MetricDefinitionValue.jsx

```diff
@@ -18,20 +18,13 @@
  */
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Metric } from '@superset-ui/chart-controls/lib/types';
 import columnType from './columnType';
 import AdhocMetricOption from './AdhocMetricOption';
 import AdhocMetric from './AdhocMetric';
 import savedMetricType from './savedMetricType';
-import adhocMetricType from './adhocMetricType';
 
 const propTypes = {
-  option: PropTypes.oneOfType([
-    savedMetricType,
-    adhocMetricType,
-    Metric,
-    PropTypes.string,
-  ]).isRequired,
+  option: PropTypes.oneOfType([PropTypes.object, PropTypes.string]).isRequired,
```

Comment:
This is just a quick fix to appease CI. Long-term solution is to migrate this file (and many others) to TypeScript and clean up/consolidate `Metric`/`AdhocMetric` related types.